app-ios/Modules/Sources/Model/Extension/Bundle.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  Bundle.swift
+//
+//
+//  Created by rei.nakaoka on 2023/08/16.
+//
+
```

Comment:
Please remove this file header 🙏🏼 
I'll add this SwiftLint rule later 🙇🏼

Reply:
I fixed at https://github.com/DroidKaigi/conference-app-2023/pull/534/commits/ced0096b78d06155e5efe9be2fb66fa9045cfd1d !